src/main/scala/djinni/PythonGenerator.scala

```diff
@@ -202,11 +202,10 @@ class PythonGenerator(spec: Spec) extends Generator(spec) {
     }
 
     tm.base match {
-      case MList => {
-        writeChecksForContainer(tm.args(0), exclude, w)
-      }
+      case MList =>
+        writeChecksForContainer(tm.args.head, exclude, w)
       case MSet | MMap => {
-        writeChecksForContainer(tm.args(0), exclude, w)
+        writeChecksForContainer(tm.args.head, exclude, w)
```

Comment:
Just ignore these kind of changes, they are recommendations by IntelliJ IDEA. I guess this PR wasn't bloated enough already so I added them... 🤷